src/docs/testing/code-debugging.md

```diff
@@ -121,7 +121,7 @@ current state or events to the console (using `debugPrint`).
 To dump the state of the Widgets library, call [`debugDumpApp()`][].
 You can call this more or less any time that the application is not in
 the middle of running a build phase (in other words, not anywhere inside a
-`build()` method), so long as the application has built at least once
+`build()` method), so long as the application has built at least once in debug mode.
```

Comment:
We need a Flutter-knowledgeable person to review this, but I can see a punctuation problem (two periods in this sentence), and it might be unclear grammatically. Maybe:

```suggestion
`build()` method), if the app has built at least once and is in debug mode
```

Reply:
The suggestion looks good.

Reply:
Thank you, @theacodes and @Piinks!